src/services/exile_service.py

```diff
@@ -104,3 +110,25 @@ async def unexile_user(
     logging_embed.set_footer(text=f"Exile ID: {exile_id}")
 
     log_info_and_embed(logging_embed, logger, f"<@{user.id}> was successfully unexiled")
+
+
+async def get_user_exiles(logging_embed: discord.Embed, user: discord.User) -> str:
+    db_user = users_database.get_user(user.id)
+    if db_user is None:
+        return "User not found in database"
+
+    exile_list = exiles_database.get_user_exiles(db_user.user_id)
+
+    if len(exile_list) == 0:
+        return "No exiles found for user"
+
+    # TODO convert this to an object for better stringifying
+    res = "Exiles found for the given user:\n"
+
+    for exile in exile_list:
+        res = (
+            res
+            + f"ID: `{exile[0]}`| Reason: `{exile[1]}` | start date: `{exile[2]}` | end date: `{exile[3]}` | status: `{ExileStatus(exile[4]).name}`"
```

Comment:
seems like https://stackoverflow.com/questions/63565825/how-to-make-data-to-be-shown-in-tabular-form-in-discord-py is a good resource for forming tables

Reply:
I updated the formatting to put this into a table. The code to handle longer reasons is a bit messy, we can truncate after a handful of characters if we want. I also think the way we're configured currently, we never expect to have more than 1 exile for a given user, so we possibly can better handle this situation

Reply:
yeah since we're not expecting the more than 1 exile situation we should probably clean this up a little bit to assume that

Reply:
I've updated this to only show the first exile, as well as cleaned up the code and moves some stuff to be demarcated as constants